redbot/core/commands/help.py

```diff
@@ -123,40 +124,55 @@ async def send_help(self, ctx: Context, help_for: HelpTarget = None):
 
         For most cases, you should use this and only this directly.
         """
+
+        help_config = await HelpSettings.from_context(ctx)
```

Comment:
This should be `help_settings` to avoid confusion.

This comment refers to every place where you used `help_config` var name for storing `HelpSettings`.